apps/teams-test-app/e2e-test-data/dialogCard.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It's reasonable to have those changes and separate each sub capability as a json file. But do you have a corresponding change in web hub sdk to retrieve those files and do test? Could you ping me for that? I would like to run it in local unless you told me that you have checked it in.

Reply:
Don't I have to merge this PR in before I use it in host sdk? E2e test command does not seem to load data from my local teams-js. I will merge this and put a PR on host sdk side.

Reply:
It seems like you are just editing UI so I would say YOU DON'T HAVE TO FOLLOW THE WAY AS BUILDING A NEW CAPABILITY, in the other words, yes, you will need this PR first and then make corresponding change in web host sdk.